snuba/consumers/snapshot_worker.py

```diff
@@ -13,50 +14,56 @@
 logger = logging.getLogger("snuba.snapshot-consumer")
 
 
-class SnapshotAwareWorker(ConsumerWorker):
-    """
-    This worker consumes messages in a similar way to its parent class ConsumerWorker.
-    The main difference is that it is used when catching up on the main topic after
-    a snapshot has been imported into clickhouse.
-
-    When we import a snapshot from postgres into clickhouse, we should apply only the
-    transactions that we find in Kafka and that are not part of the snapshot (which means
-    that they were produced after the snapshot was taken).
+CDCEvent = Mapping[str, Any]  # TODO: Replace with ``TypedDict``
```

Comment:
Will deal with this later once `MessageProcessor` is generic on `TPayload` and this can be enforced. In a holding pattern on that change as it will have minor conflicts with #1192 (which will also benefit from the processor genericization) and would like to see that land first.